### DISCUSSION TITLE
Trigger change mode

### DISCUSSION THREAD

**Discussion by LeulAria** [react-hook-form/react-hook-form#10384] [Q&A] [answered]
So basically when you submit a form using
`onSubmit` mode after submitting the validation will become onChange mode validation.
how to achieve this in trigger function.

**Comment by Moshyfawn** [comment, marked as answer]
Would you be able to expand on your use-case? Have you tried [the onTouched mode](https://react-hook-form.com/api/useform/#mode)

**Reply by LeulAria** [reply]
yes  [the onTouched mode](https://react-hook-form.com/api/useform/#mode) works nice didn't tried it.
im building a stepper i think using trigger with like.

```ts
step_forms = [ ['name','email'], ['username', 'password]]
trigger(step_form[i])
```

so it's working but it do onChange validation on dirtyInputs for underty it use onBlur validation i guess it work.
